src/test/java/uk/gov/ons/ssdc/caseprocessor/messaging/RefusalReceiverIT.java

```diff
@@ -29,16 +29,18 @@
 import uk.gov.ons.ssdc.caseprocessor.model.repository.EventRepository;
 import uk.gov.ons.ssdc.caseprocessor.model.repository.UacQidLinkRepository;
 import uk.gov.ons.ssdc.caseprocessor.model.repository.WaveOfContactRepository;
-import uk.gov.ons.ssdc.caseprocessor.utils.QueueSpy;
-import uk.gov.ons.ssdc.caseprocessor.utils.RabbitQueueHelper;
+import uk.gov.ons.ssdc.caseprocessor.testutils.QueueSpy;
+import uk.gov.ons.ssdc.caseprocessor.testutils.RabbitQueueHelper;
 
 @ContextConfiguration
 @ActiveProfiles("test")
 @SpringBootTest
 @RunWith(SpringJUnit4ClassRunner.class)
 public class RefusalReceiverIT {
   private static final UUID TEST_CASE_ID = UUID.randomUUID();
-  private static final String RH_CASE_QUEUE = "case.rh.case";
+
+  @Value("${queueconfig.outbound-case-queue}")
```

Comment:
I don't agree with putting stuff into `application.yml` that's just for testing. If we need to point the prod code at test resources, we can use `application-test.yml` but it makes it confusing to tell the difference between prod config and test stuff which has no relevance to prod if we mix it together

Reply:
use a `TestConstants` class if you want to move this kind of stuff out of the specific test

Reply:
I was just trying to get away from having the same queue names redefined in many different files, if you think a `TestConstants` files makes more sense I'll do that. The queues purely used in the tests are already in the `application-test.yml`, not `application.yml` though.